src/features/home/Home.tsx

```diff
@@ -82,26 +90,30 @@ export class Home extends React.Component<Props, State> {
     this.props.navigation.dispatch(DrawerActions.toggleDrawer());
   };
 
-  private keyExtractorType: (item) => string = (item) => `Type - ${item.id}`;
-  private keyExtractorDish: (item) => string = (item) => `Dish - ${item.id}`;
+  private keyExtractorType: (item: {id: number; type: TypesDish}) => string = (item) => `Type - ${item.id}`;
+  private keyExtractorDish: (item: DishModel) => string = (item) => `Dish - ${item.id}`;
 
   private changeType: (currentType: TypesDish) => void = (currentType) => {
     this.setState({currentType});
   };
 
-  private navigateDish: (dish: Dish) => void = (dish) => {
-    this.props.navigation.navigate(RootScreens.Dish, {dish});
+  private navigateDish: (dish: DishModel) => void = (dish) => {
+    this.props.navigation.navigate(RootScreens.Dish, {dish, onPress: this.addDishInBasket});
   };
 
   private navigateSearch: () => void = () => {
```

Comment:
@DiamondRock597 No need to describe method interface with new eslint rules version.